From b0e764983e19637290090ccfc58b93ccadbd251a Mon Sep 17 00:00:00 2001
From: Daryl Denson <daryldenson0405@gmail.com>
Date: Sat, 10 Oct 2026 01:52:42 +0200
Subject: [PATCH] zuma: install Pixel default permissions as
 default-permissions-zuma.xml

Evolution X's vendor/gms installs its own product/etc/default-permissions/
default-permissions.xml. Both are needed and PermissionController reads every
file in that directory, so give zuma's its own name.
---
 common.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/common.mk b/common.mk
index 8b1cb1ab..305c9f3d 100644
--- a/common.mk
+++ b/common.mk
@@ -213,7 +213,7 @@ PRODUCT_PACKAGES += \
 $(call inherit-product, $(SRC_TARGET_DIR)/product/core_64_bit_only.mk)
 
 PRODUCT_COPY_FILES += \
-	device/google/zuma/default-permissions.xml:$(TARGET_COPY_OUT_PRODUCT)/etc/default-permissions/default-permissions.xml \
+	device/google/zuma/default-permissions.xml:$(TARGET_COPY_OUT_PRODUCT)/etc/default-permissions/default-permissions-zuma.xml \
 	device/google/zuma/component-overrides.xml:$(TARGET_COPY_OUT_VENDOR)/etc/sysconfig/component-overrides.xml \
 	frameworks/native/data/etc/handheld_core_hardware.xml:$(TARGET_COPY_OUT_VENDOR)/etc/permissions/handheld_core_hardware.xml \
 
-- 
2.43.0

